### DISCUSSION TITLE
secretflow组件添加失败

### DISCUSSION THREAD

**Discussion by 123abcs** [secretflow/secretflow#1565] [Q&A]
添加secretflow组件，运行注册docker镜像命令时报错如下
ERROR: /root/.cache/bazel/_bazel_root/2e9839e637bbe387e275e64e16628705/external/com_github_gabime_spdlog/BUILD.bazel:24:20: Foreign Cc - CMake: Building spdlog failed: (Exit 1): bash failed: error executing command (from target @com_github_gabime_spdlog//:spdlog) /bin/bash -c bazel-out/k8-opt/bin/external/com_github_gabime_spdlog/spdlog_foreign_cc/wrapper_build_script.sh

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
Target //secretflow_lib/binding:_lib.so failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /home/admin/src_copied/secretflow_lib/binding/BUILD.bazel:27:17 Linking secretflow_lib/binding/_lib.so failed: (Exit 1): bash failed: error executing command (from target @com_github_gabime_spdlog//:spdlog) /bin/bash -c bazel-out/k8-opt/bin/external/com_github_gabime_spdlog/spdlog_foreign_cc/wrapper_build_script.sh

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
INFO: Elapsed time: 59.249s, Critical Path: 27.66s
INFO: 49 processes: 12 internal, 37 linux-sandbox.
FAILED: Build did NOT complete successfully
error: command '/usr/bin/bazel' failed with exit code 1
请问如何解决呢，谢谢

**Comment by john8628** [comment]
参考https://github.com/secretflow/spu/blob/main/CONTRIBUTING.md#build 这个文档，或者参考这里面的dockerfile（https://github.com/secretflow/devtools/tree/main/dockerfiles）安装相关依赖组件
